packages/flutter_nekoton_bridge/js/src/index.ts

```diff
@@ -0,0 +1,43 @@
+;(window as Record<string, any>).__hasEverscaleProvider = true
+
+// TODO: remove in future
+;(window as Record<string, any>).hasTonProvider = true
+
+import { Buffer } from 'buffer';
+window.Buffer = Buffer;
+
+// @ts-ignore
+let __define: Define | undefined
+
+const cleanContextForImports = () => {
+    // @ts-ignore
```

Comment:
BUT we have "tabWidth: 2" in .prettierrc! Seems like .prettierrc is only for decorations.